aiida/orm/entities.py

```diff
@@ -172,7 +172,7 @@ def objects(cls: EntityType) -> Collection[EntityType]:  # pylint: disable=no-se
 
         :return: an object that can be used to access entities of this type
         """
-        warn_deprecation('This property is deprecated, use `.collection` instead.', version=3, stacklevel=2)
+        warn_deprecation('`objects` property is deprecated, use `collection` instead.', version=3, stacklevel=2)
```

Comment:
I wanted to use the f`{cls.__name__}` here as well, but mypy complains that Entity does not have the `__name__` attribute.

Reply:
Does it actually work though? I think it should work as `Entity` is a normal class and so should have a `__name__` just fine. See if it works and if so, just add it, I think it would make the message even clearer and then just add a `# type: ignore[error-message]` comment on the line

Reply:
Also, maybe the stacklevel should be adjusted? Did you get to see where in the stack it was being called? Maybe increase the stacklevel to see if you can get the exact call location to show up